crossdock/client/errorstchout/behavior.go

```diff
@@ -150,6 +152,27 @@ func Run(t crossdock.T) {
 				assert.Equal(tchannel.ErrCodeUnexpected, code, "must produce transport error")
 			},
 		},
+		{
+			name:      "remote timeout error",
+			procedure: "waitfortimeout/raw",
+			body:      []byte("{}"),
+			headers:   []byte{0x0, 0x0},
+			encoding:  "raw",
+			validate: func(res3 []byte, isAppErr bool, err error) {
+				assert.Error(err, "is error")
+				assert.False(isAppErr, "waitfortimeout/raw procedure must not produce application error")
```

Comment:
Same here: If this returns false, then the cast on the next line will panic so short circuit early. 

```go
if !assert.False(...) { return }
```